simulation/agents.py

```diff
@@ -351,20 +377,29 @@ def update_heating_status(self, model: "CnzAgentBasedModel") -> None:
         if random.random() < proba_failure:
             self.heating_functioning = False
 
-    def compute_heat_pump_capacity_kw(self, heat_pump_type: HeatingSystem) -> float:
+    def compute_heat_pump_capacity_kw(self, heat_pump_type: HeatingSystem) -> int:
 
         capacity_kw = (
             HEAT_PUMP_CAPACITY_SCALE_FACTOR[heat_pump_type] * self.floor_area_sqm
         )
-        return min(
-            max(capacity_kw, MIN_HEAT_PUMP_CAPACITY_KW[heat_pump_type]),
-            MAX_HEAT_PUMP_CAPACITY_KW[heat_pump_type],
+        return math.ceil(
```

Comment:
Also modified in this PR so that household is assigned the higher capacity where in-between bands, and to return an `int` that can be used in a lookup dictionary of heat pump costs (`MEDIAN_COST_GBP_HEAT_PUMP_AIR_SOURCE`, `MEDIAN_COST_GBP_HEAT_PUMP_GROUND_SOURCE`)